README.md

```diff
@@ -1,19 +1,20 @@
-# style-snooper
-
-A simple tool to extract the WPF styles for `FrameworkElement`s.
-
-These can provide a useful starting point to your own customisations.
-
-![screenshot](https://raw.githubusercontent.com/drewnoakes/style-snooper/master/screenshot.png)
-
-This repo derives from code source from [Eli Arbel](https://arbel.net/2006/11/05/revamped-style-snooper/),
-though the original source may have been [Walt Tirscher](https://wpfwonderland.wordpress.com/2007/01/02/wpf-tools-stylesnooper/).
-These various projects were hosted on blogs which have since yielded broken links to downloads and such.
-Hopefully this repo lasts as long as WPF developers do.
-
-[Changes in this repo](https://github.com/drewnoakes/style-snooper/compare/b8529f67ff5cd6b2b6da07dd0bf5e9f053049810...master) include:
-
-- Simplification of output XAML
-- Upgrade from .NET 3.0 to .NET 4.5
-- UI layout improvements
-- Code style and project clean up
+# style-snooper
+
+A simple tool to extract the WPF styles for `FrameworkElement`s.
+
+These can provide a useful starting point to your own customisations.
+
+![screenshot](https://raw.githubusercontent.com/drewnoakes/style-snooper/master/screenshot.png)
+
+This repo derives from code source from [Eli Arbel](https://arbel.net/2006/11/05/revamped-style-snooper/),
+though the original source may have been [Walt Tirscher](https://wpfwonderland.wordpress.com/2007/01/02/wpf-tools-stylesnooper/).
+These various projects were hosted on blogs which have since yielded broken links to downloads and such.
+Hopefully this repo lasts as long as WPF developers do.
+
+[Changes in this repo](https://github.com/drewnoakes/style-snooper/compare/b8529f67ff5cd6b2b6da07dd0bf5e9f053049810...master) include:
+
+- Simplification of output XAML
+- Upgrade from .NET 3.0 to .NET 4.5
+- UI layout improvements
+- Code style and project clean up
+- Added an icon. Credits: [CC-BY Lucas Helle, CL](https://thenounproject.com/search/?q=Nose&i=3608372), modified by Wolfgang Kleinschmit.

```

Comment:
The diff here is huge. Maybe due to a change in line endings? I've added a `.gitattributes` file that should standardise line endings in future to prevent this.

Reply:
Well, that made this have conflicts. Line endings in git are such a pain.